### PR TITLE
Fix the security concerns relating to plain text passwords

### PR DESCRIPTION
**Pull request by sac**

### CODE REVIEW

**Comment by rohantmp** [comment]
+1 looks good

File: playbooks/hc-ansible-deployment/he_gluster_vars.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  // The passwords variables should be encryped using ansible-vault
```
Comment (rohantmp) [inline playbooks/hc-ansible-deployment/he_gluster_vars.json]: json does not support comments, should we be including this here?

File: playbooks/hc-ansible-deployment/he_gluster_vars.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
   "he_appliance_password": "root-password-of-host",
```
Comment (rohantmp) [inline playbooks/hc-ansible-deployment/he_gluster_vars.json]: perhaps we could include the encryption warning in the dummy password.

Comment (sac) [reply playbooks/hc-ansible-deployment/he_gluster_vars.json]: @rohantmp no. We to somehow convey the user that it has to be encrypted. Is there any way at all to add comments in json? Like "_comment": ... something like that?

**Pull request merged**